trending_data/src/main/java/com/google/sps/data/TwitterResponse.java

```diff
@@ -0,0 +1,9 @@
+package com.google.sps.data;
+
+public class TwitterResponse {
+  private TrendingTopic[] trends;
+
+  public TrendingTopic[] getTrends() {
```

Comment:
If you can avoid it, it's better to drop the arrays as much as possible and use Collections instead (item 28 in Effective Java). On top of that, if this TrendingTopic[] is not going to be modified, the best practice is to use immutable collections (item 17).
You can do this in one line by returning a List<TrendingTopic> with `Collections.unmodifiableList(Arrays.asList(trends));`